### ISSUE TITLE
/transactions/address/../limit/1001+

### ISSUE THREAD

**Issue by demon2051** [wavesplatform/Waves#38]
/transactions/address/../limit can only return 1000 last records. if limit is bigger than 1000 node responding with 
`{"error": 10, "message": "Too big sequences requested"}`

So if account have, for example, 2000 transactions and we need to get oldest txs /transactions/address/../limit/ became useless.

**Comment by bcdev-** [comment]
Yes, currently there is no way to get all transactions for an account. :-/
We need a proper way to return {start,start+limit}.

**Comment by javichi** [comment]
+1

I understand there is a limit for performance reasons. However there should be at least a way to paginate through all your transactions when you have more than that limit
